Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.yml

```diff
@@ -1602,15 +1602,207 @@ script:
+    name: msg-get-last-estimate-statistics-operation
+  - arguments:
+      - description: The unique ID of the eDiscovery case.
+        isArray: false
+        name: case_id
+        required: true
+      - description: The unique ID of the eDiscovery search.
+        isArray: false
+        name: search_id
+        required: true
+      - description: Items to include in the export (e.g., "searchHits", "partiallyIndexed").
+        isArray: false
+        name: export_criteria
+        required: true
+        auto: PREDEFINED
+        predefined:
+          - searchHits
+          - partiallyIndexed
+      - description: Comma-separated list of additional options (e.g., "cloudAttachments,allDocumentVersions"). The advancedIndexing value is only available if exportCriteria includes partiallyIndexed.
+        isArray: false
+        name: additional_options
+        required: true
+        auto: PREDEFINED
+        predefined:
+          - none
+          - teamsAndYammerConversations
+          - cloudAttachments
+          - allDocumentVersions
+          - subfolderContents
+          - listAttachments
+          - htmlTranscripts
+          - advancedIndexing
+          - allItemsInFolder
+      - description: Optional display name for the export report.
+        isArray: false
+        name: display_name
+        required: false
+      - description: Optional description for the export report.
+        isArray: false
+        name: description
+        required: false
+      - description: Optional export location for partially indexed items. The nonresponsiveLocations value is only available if exportCriteria includes partiallyIndexed.
+        isArray: false
+        name: export_location
+        required: false
+        auto: PREDEFINED
+        predefined:
+          - responsiveLocations
+          - nonresponsiveLocations
+      - description: Optional cloud attachment version.
+        isArray: false
+        name: cloud_attachment_version
+        required: false
+        auto: PREDEFINED
+        defaultValue: latest
+        predefined:
+          - latest
+          - recent10
+          - recent100
+          - all
+      - description: Optional document version.
+        isArray: false
+        name: document_version
+        required: false
+    outputs:
+      - contextPath: MsGraph.eDiscovery.ExportReport.CaseID
+        description: The eDiscovery case ID.
+        type: String
+      - contextPath: MsGraph.eDiscovery.ExportReport.SearchID
+        description: The eDiscovery search ID.
+        type: String
+      - contextPath: MsGraph.eDiscovery.ExportReport.OperationURL
+        description: The operation URL to poll export status.
+        type: String
+    description: Starts an eDiscovery export report for a given case and search in Microsoft Purview and returns the operation URL for polling. Use the operation URL with 'msg-get-ediscovery-report' to check the status or download the report once ready.
```

Comment:
are there any details here of what a report can contain? Or if its too much info, can we point here to msft docs?